cmd/root.go

```diff
@@ -143,62 +198,57 @@ any client SSL certificates.`,
 
 	cmd.RunE = func(*cobra.Command, []string) error { return runSignalWrapper(c) }
 
+	// Override Cobra's default messages.
+	cmd.PersistentFlags().BoolP("help", "h", false, "Display help information for cloudsql-proxy")
+	cmd.PersistentFlags().BoolP("version", "v", false, "Print the cloudsql-proxy version")
+
 	// Global-only flags
 	cmd.PersistentFlags().StringVarP(&c.conf.Token, "token", "t", "",
-		"Bearer token used for authorization.")
+		"Use bearer token as a source of IAM credentials.")
 	cmd.PersistentFlags().StringVarP(&c.conf.CredentialsFile, "credentials-file", "c", "",
-		"Path to a service account key to use for authentication.")
+		"Use service account key file as a source of IAM credentials.")
 	cmd.PersistentFlags().BoolVarP(&c.conf.GcloudAuth, "gcloud-auth", "g", false,
-		"Use gcloud's user configuration to retrieve a token for authentication.")
+		"Use gcloud's user credentials as a source of IAM credentials.")
 	cmd.PersistentFlags().BoolVarP(&c.conf.StructuredLogs, "structured-logs", "l", false,
-		"Enable structured logs using the LogEntry format")
+		"Enable structured logging with LogEntry format")
 	cmd.PersistentFlags().Uint64Var(&c.conf.MaxConnections, "max-connections", 0,
-		`Limits the number of connections by refusing any additional connections.
-When this flag is not set, there is no limit.`)
+		"Limit the number of connections. Default is no limit.")
 	cmd.PersistentFlags().DurationVar(&c.conf.WaitOnClose, "max-sigterm-delay", 0,
-		`Maximum amount of time to wait after for any open connections
-to close after receiving a TERM signal. The proxy will shut
-down when the number of open connections reaches 0 or when
-the maximum time has passed. Defaults to 0s.`)
+		"Maximum time to wait for connections to close after receiving a TERM signal.")
 	cmd.PersistentFlags().StringVar(&c.telemetryProject, "telemetry-project", "",
-		"Enable Cloud Monitoring and Cloud Trace integration with the provided project ID.")
+		"Enable Cloud Monitoring and Cloud Trace with the provided project ID.")
 	cmd.PersistentFlags().BoolVar(&c.disableTraces, "disable-traces", false,
-		"Disable Cloud Trace integration (used with telemetry-project)")
+		"Disable Cloud Trace integration (used with --telemetry-project)")
 	cmd.PersistentFlags().IntVar(&c.telemetryTracingSampleRate, "telemetry-sample-rate", 10_000,
-		"Configure the denominator of the probabilistic sample rate of traces sent to Cloud Trace\n(e.g., 10,000 traces 1/10,000 calls).")
+		"Configure the sample rate of traces sent to Cloud Trace. A smaller number means more traces.")
```

Comment:
"Set the Cloud Trace sample rate. A number 1 and 10000, smaller means more traces."